bodhi/server/consumers/automatic_updates.py

```diff
@@ -141,10 +141,15 @@ def __call__(self, message: fedora_messaging.api.Message) -> None:
                 stable_karma=3,
                 unstable_karma=-3,
                 user=user,
+                status=UpdateStatus.testing,
             )
 
-            log.debug("Setting request for new update.")
-            update.set_request(dbsession, UpdateRequest.testing, owner_name)
+            # Comment on the update that it was automatically created.
+            update.comment(
+                dbsession,
+                str("This update was automatically created"),
```

Comment:
This was merged but I think the comment string should have been stored in the config, so that it can be easily changed without the need to create a new release for Bodhi.

Reply:
I thought about it but thought this can be tweaked/changed later.